src/qrules/io/_dot.py

```diff
@@ -331,28 +332,40 @@ def __render_fraction(value: Any, plusminus: bool) -> str:
             if plusminus:
                 return f"{nom:+}/{denom}"
             return f"{nom}/{denom}"
+    if isinstance(value, Fraction):
+        return _render_fraction(value, plusminus)
     if plusminus:
         return f"{value:+}"
     return str(value)
 
 
+def _render_fraction(fraction: Fraction, plusminus: bool = False) -> str:
+    if fraction.denominator == 1:
+        if plusminus and fraction.numerator > 0:
+            return f"{fraction.numerator:+}"
+        return str(fraction.numerator)
+    if plusminus:
+        return f"{fraction.numerator:+}/{fraction.denominator}"
+    return f"{fraction.numerator}/{fraction.denominator}"
```

Comment:
Maybe this can be simplified with the fact that

```python
>>> f"{Fraction(1, 2)}"
'1/2'
```

Reply:
Only a small change, as 
```python
f"{fraction:+}"
```
doesn't work.